doc/changelog.md

```diff
@@ -0,0 +1,923 @@
+    ([SmartSim-PR141](https://github.com/CrayLabs/SmartSim/pull/141))
+-   Update SmartSim Examples
+    ([SmartSim-PR68](https://github.com/CrayLabs/SmartSim/pull/68))
+    ([SmartSim-PR100](https://github.com/CrayLabs/SmartSim/pull/100))
+
+### 0.3.2
+
+Released on August 10, 2021
+
+Description:
+
+-   Upgraded RedisAI backend to 1.2.3
+    ([SmartSim-PR69](https://github.com/CrayLabs/SmartSim/pull/69))
+-   PyTorch 1.7.1, TF 2.4.2, and ONNX 1.6-7
+    ([SmartSim-PR69](https://github.com/CrayLabs/SmartSim/pull/69))
+-   LSF launcher for IBM machines
+    ([SmartSim-PR62](https://github.com/CrayLabs/SmartSim/pull/62))
+-   Improved code coverage by adding more unit tests
+    ([SmartSim-PR53](https://github.com/CrayLabs/SmartSim/pull/53))
+-   Orchestrator methods to get address and check status
+    ([SmartSim-PR60](https://github.com/CrayLabs/SmartSim/pull/60))
+-   Added Manifest object that tracks deployables in Experiments
+    ([SmartSim-PR61](https://github.com/CrayLabs/SmartSim/pull/61))
+-   Bug fixes
+    ([SmartSim-PR52](https://github.com/CrayLabs/SmartSim/pull/52))
+    ([SmartSim-PR58](https://github.com/CrayLabs/SmartSim/pull/58))
+    ([SmartSim-PR67](https://github.com/CrayLabs/SmartSim/pull/67))
+    ([SmartSim-PR73](https://github.com/CrayLabs/SmartSim/pull/73))
+-   Updated documentation and examples
+    ([SmartSim-PR51](https://github.com/CrayLabs/SmartSim/pull/51))
+    ([SmartSim-PR57](https://github.com/CrayLabs/SmartSim/pull/57))
+    ([SmartSim-PR71](https://github.com/CrayLabs/SmartSim/pull/71))
+-   Improved IP address aquisition
+    ([SmartSim-PR72](https://github.com/CrayLabs/SmartSim/pull/72))
+-   Binding database to network interfaces
+
+### 0.3.1
+
+Released on May 5, 2021
+
+Description: This release was dedicated to making the install process
+easier. SmartSim can be installed from PyPI now and the `smart` cli tool
+makes installing the machine learning runtimes much easier.
+
+-   Pip install
+    ([SmartSim-PR42](https://github.com/CrayLabs/SmartSim/pull/42))
+-   `smart` cli tool for ML backends
+    ([SmartSim-PR42](https://github.com/CrayLabs/SmartSim/pull/42))
+-   Build Documentation for updated install
+    ([SmartSim-PR43](https://github.com/CrayLabs/SmartSim/pull/43))
+-   Migrate from Jenkins to Github Actions CI
+    ([SmartSim-PR42](https://github.com/CrayLabs/SmartSim/pull/42))
+-   Bug fix for setup.cfg
+    ([SmartSim-PR35](https://github.com/CrayLabs/SmartSim/pull/35))
+
+### 0.3.0
+
+Released on April 1, 2021
+
+Description:
+
+-   initial 0.3.0 (first public) release of SmartSim
+
+------------------------------------------------------------------------
+
+## SmartRedis changelog
+
+```{eval-rst}
+.. include:: ../smartredis/doc/changelog.md
+   :parser: myst_parser.sphinx_
+```
+
+------------------------------------------------------------------------
+
+## SmartDashboard changelog
+
+```{eval-rst}
+.. include:: ../smartdashboard/doc/changelog.md
+   :parser: myst_parser.sphinx_
+```
```

Comment:
Curious about what I'm supposed to see down here? When I preview the changelog it's just these lines in a code block without any parsed info.

Reply:
yes yes so I am using myst_parser which lets me using a combination of markdown and rst and its an extension of sphinx - so it just lets me include md files with rst files. and here Im using an eval directive which allows me to include content from a specific file, and I specify using :parser: that I want the file to be parsed using myst_parser, otherwise it just prints everything raw. So this snippet just lets me incorporate md into sphinx documentation.

Reply:
Thank you so much for the explanation! Okay that makes a lot more sense to me now. So viewing the raw `.md` file will not have the `SmartRedis` and `SmartDashboard` info parsed out, but the built documentation on `readthedocs` will include all that. Makes sense!